src/components/sbb-calendar/sbb-calendar.tsx

```diff
@@ -0,0 +1,420 @@
+    return (
+      <table class="sbb-calendar__table">
+        <thead class="sbb-calendar__table-header">
+          <tr class="sbb-calendar__table-header-row">{this._createTableHeader()}</tr>
+        </thead>
+        <tbody class="sbb-calendar__table-body">{this._createTableBody(weeks)}</tbody>
+      </table>
+    );
+  }
+
+  /** Creates the table header with the months header cells. */
+  private _createTableHeader(): JSX.Element {
+    return this._weekdays.map((day: Weekday) => (
+      <th>
+        <span class="visually-hidden">{day.long}</span>
+        <span aria-hidden="true">{day.narrow}</span>
+      </th>
+    ));
+  }
+
+  /**
+   * Creates the table body with the days cells. For the first row, it also considers the possible day's offset.
+   */
+  private _createTableBody(weeks: Day[][]): JSX.Element {
+    return weeks.map((week: Day[], rowIndex: number) => {
+      const firstRowOffset: number = NativeDateAdapter.DAYS_PER_WEEK - week.length;
+      if (rowIndex === 0 && firstRowOffset) {
+        return (
+          <tr>
+            <td
+              colSpan={firstRowOffset}
+              data-day={`0 ${week[0].monthValue} ${week[0].yearValue}`}
+            ></td>
+            {this._createDayCells(week)}
+          </tr>
+        );
+      }
+      return <tr>{this._createDayCells(week)}</tr>;
+    });
+  }
+
+  /** Creates the cells for the days. */
+  private _createDayCells(week: Day[]): JSX.Element {
+    const today: string = this._dateAdapter.today().toISOString();
+    return week.map((day: Day) => {
+      const isOutOfRange = !this._isDayInRange(day.value);
+      const isFilteredOut = !this.dateFilter(new Date(day.value));
+      const selected: boolean = this._selected && day.value === this._selected;
+      const dayValue = `${day.dayValue} ${day.monthValue} ${day.yearValue}`;
+      return (
+        <td>
+          <button
+            class={{
+              'sbb-datepicker__day': true,
+              'sbb-datepicker__day-today': day.value === today,
+              'sbb-datepicker__day-selected': selected,
+              'sbb-datepicker__crossed-out': !isOutOfRange && isFilteredOut,
+            }}
+            onClick={(event) => this._selectDate(day.value, event)}
+            disabled={isOutOfRange || isFilteredOut}
+            aria-label={this._dateAdapter.getAccessibilityFormatDate(day.value)}
+            aria-pressed={String(selected)}
+            aria-disabled={String(isOutOfRange || isFilteredOut)}
+            data-day={dayValue}
+            tabindex="-1"
+            onKeyDown={(evt: KeyboardEvent) => this._handleKeyboardEvent(evt, day)}
+          >
+            {day.dayValue}
+          </button>
+        </td>
+      );
+    });
+  }
+
+  /** Creates the month label. */
+  private _createMonthLabel(d: Date): JSXElement {
+    return (
+      <span class="sbb-calendar__controls-month-label">
+        {this._dateAdapter.getMonthNames('long')[this._dateAdapter.getMonth(d)]}{' '}
```

Comment:
instead of the fancy  {' '} we could concatenate to string with template string?